### PR TITLE
Refactor stat tracker

### PR DESCRIPTION
**Pull request by kevxo**
Created all the test for stat tracker.

### CODE REVIEW

**Comment by ShaunDaneJames** [comment]
Good job on making the tests for stat tracker.

File: lib/team_manager.rb [hunk 1]
```diff
@@ -167,20 +167,13 @@ def fewest_tackles(season)
...
-
   def team_info(team_id)
     hash = {}
     @team_data.each do |team|
       if team_id == team.team_id.to_s
         hash['team_id'] = team.team_id.to_s
-        hash['franchiseid'] =  team.franchiseid.to_s
-        hash['teamname'] = team.teamname
+        hash['franchise_id'] =  team.franchiseid.to_s
+        hash['team_name'] = team.teamname
```
Comment (carson-jardine) [inline lib/team_manager.rb:176]: good catch!

**Review by carson-jardine — APPROVED** [review]

**Comment by carson-jardine** [comment]
LGTM!!! Thanks for compiling all this, Kevin!

**Comment by Oxalisviolacea** [comment]
This looks great. Thanks for all your hard work!

**Pull request merged**